src/components/pages/LoginPage.vue

```diff
@@ -0,0 +1,287 @@
+                            type="text" 
+                            name="email"
+                            placeholder="Email" 
+                            id="email" 
+                            class="form-input" 
+                            autocomplete="false"
+                            v-bind:class="{
+                              'is-valid' : validation.valid.email,
+                              'is-invalid' : validation.invalid.email
+                            }"
+                            v-on:focus="validation('email')"
+                            v-model="email"
+                            required
+                          />
+                </div>
+
+
+                <!-- Valid Password -->
+                <span class="invalid-feedback" v-if="validation.invalid.password">
+                    {{ validation.invalid.password }}
+                </span>
+                <!-- Invalid password -->
+                <span class="valid-feedback" v-if="validation.valid.password">
+                    {{ validation.valid.password }}
+                </span>
+                <div class="form-field">
+                    <label class="lock" for="login-password"><span class="hidden">Password</span></label>
+                        <input 
+                            id="password" 
+                            type="password"
+                            name="password"
+                            class="form-input" 
+                            placeholder="Password" 
+                            v-bind:class="{
+                                'is-valid' : validation.valid.password,
+                                'is-invalid' : validation.invalid.password
+                            }"
+                            v-on:focus="validation('password')"
+                            v-model="password"
+                            required
+                        />
+                </div>
+                          
+                <div class="form-field"> 
+                    <input type="submit" value="Log in">
+                </div>
+                    <p class="register-link">Don't have account? 
+                        <router-link to="/register"><span class="link">Register here</span></router-link>
+                    </p>
+                </form>
+                </div>
+            </div>
+        </LayoutDiv>
+    </div>
+</template>
+
+<script>
+import CustomHeader from '@/components/shared/CustomHeader.vue';
+import NavBar from '@/components/shared/NavBar.vue';
+import LayoutDiv from '@/components/shared/LayoutDiv.vue';
+
+export default {
+  components: {
+      CustomHeader,
+      NavBar,
+      LayoutDiv
+  },
+  data() {
+    return {
+      email: '',
+      emailBlured: false,
+      password: '',
+      passwordBlured: false,
+      valid: false,
+      submitted: false,
+
+      validation: {
+        invalid: {},
+        valid: {}
+      }
+    };
+  },
+
+  methods: {
```

Comment:
You need to add a method for what happens when the user clicks the Submit button. The method should verify that all of the inputs have values. If all input fields are filled out, create an object that will be the payload when we have a login endpoint in the API. For now, just console.log a message saying 'Submitting login' and console.log the payload object.